CRM/MembershipExtras/Form/MembershipPeriod/PeriodRules.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+use CRM_MembershipExtras_ExtensionUtil as E;
+
+/**
+ * Membership Period Rules form controller
+ * 
+ * @see http://wiki.civicrm.org/confluence/display/CRMDOC43/QuickForm+Reference
+ */
+class CRM_MembershipExtras_Form_MembershipPeriod_PeriodRules extends CRM_Core_Form {
```

Comment:
I think we should move this to a separate path, out of MembershipPeriod, as the form deals with settings, and not the MembershipPEriod directly. Maybe `CRM_MembershipExtras_Form_Settings_PeriodRules`?